README.md

```diff
@@ -1,24 +1,146 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Demo](#demo)
+- [💻 Presentation](#presentation)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+# 📖 BUDGET APP <a name="about-project"></a>
 
-* System dependencies
+**[Budget App]** is a project created using Ruby on Rails for users to create bills. It can also allow users to create items under these bills with their corresponding amounts. It can be used to track the spending of an individual.
 
-* Configuration
+## 🛠 Built With <a name="built-with"></a>
 
-* Database creation
+### Tech Stack <a name="tech-stack"></a>
 
-* Database initialization
+<details>
+<summary>Budget App built with Ruby on Rails</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* How to run the test suite
+### Key Features <a name="key-features"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+- **Built with Ruby on Rails**
+- **Connected to Postgresql**
+- **Devise used for authentication**
+- **Capybara used for integration tests**
 
-* Deployment instructions
+## 💻 Demo <a name="demo"></a>
+Here is the live [Demo](https://mysite-swfq.onrender.com/)
 
-* ...
+## 💻 Presentation <a name="presentation"></a>
+Here is the [Project Presentation](https://www.loom.com/share/565abfa2996249f99f9eb3586b628bdc)
+
```

Comment:
- [ ] Kindly note that it is said in the requirement of the video presentation that you should add a highlight about a piece of the code you findsinteresting or is proud of making
<img width="1039" alt="image" src="https://github.com/Uthmanbello/budget_app/assets/65068771/e74b20a1-d22a-4cbb-8f17-713b1133fbba">